### ISSUE TITLE
Module3: 异构双轨并发测算引擎 (Dual-Track Computation Engine)

### ISSUE THREAD

**Issue by nekobola** [nekobola/AE_LLM_RL_FOF#19]
# Module3: 异构双轨并发测算引擎

## 模块定位
系统的确定性数学求解器。
输入模块2提供的5x60收益率矩阵，输出常态(偏进攻)与异常(纯防守)两组5维权重向量。

## 资产索引
- 0: 宽基 (Broad)
- 1: 卫星 (Satellite)
- 2: 固收 (Fixed Income)
- 3: 避险/黄金 (Safe Haven)
- 4: 现金/货币 (Cash)

### 1. 协方差收缩
使用 sklearn.covariance.LedoitWolf 计算收缩协方差矩阵 Sigma (5x5)

### 2. 预期收益率
mu = mean(R, axis=1) 日均收益率向量 (5,)

### 3. SLSQP约束优化
目标函数: 最大化夏普比率 -> 最小化 -w.T @ mu / sqrt(w.T @ Sigma @ w)

约束:
- 等式: sum(w) = 1.0
- Bounds:
  - w[0] 宽基: [0.05, 0.40]
  - w[1] 卫星: [0.05, 0.30]
  - w[2] 固收: [0.10, 0.60]
  - w[3] 避险: [0.00, 0.20]
  - w[4] 现金: [0.00, 0.20]

### 权益切断
w[0] = 0.0, w[1] = 0.0

### Epsilon保护倒数加权
epsilon = 1e-6
iv_i = 1 / (sigma_i + epsilon) for i in [2,3,4]
w_i = iv_i / sum(iv_j for j in [2,3,4])

## 输出
return W_Normal, W_Event (两个5维numpy数组)

## 依赖
numpy, scipy.optimize, sklearn.covariance

## 预期交付物
1. src/compute/dual_track_engine.py — 双轨并发测算主逻辑
2. src/compute/normal_track.py — Markowitz优化
3. src/compute/event_track.py — 倒数波动率防御
4. config.yaml — bounds参数
